scripts/example-project.sh

```diff
@@ -8,11 +8,34 @@ echo "Initializing the project example project ${PROJECT_BASE}/rill-developer-ex
 npm run cli --silent -- init --project ${PROJECT_BASE}/rill-developer-example
 
 echo "Downloading dataset for example project..."
-curl -s http://pkg.rilldata.com/rill-developer-example/data/flightlist.zip --output ${PROJECT_BASE}/rill-developer-example/flightlist.zip
-unzip ${PROJECT_BASE}/rill-developer-example/flightlist.zip -d ${PROJECT_BASE}/rill-developer-example/
+curl -s http://pkg.rilldata.com/rill-developer-example/example-assets.zip --output ${PROJECT_BASE}/rill-developer-example/example-assets.zip
+unzip ${PROJECT_BASE}/rill-developer-example/example-assets.zip -d ${PROJECT_BASE}/rill-developer-example/
 
-echo "Importing example dataset into the project..."
-npm run cli --silent -- import-source ${PROJECT_BASE}/rill-developer-example/data/flightlist_2022_02.csv --project ${PROJECT_BASE}/rill-developer-example
+echo "Importing example datasets into the project..."
```

Comment:
In order to make this work on Windows we are not using this `.sh` script anymore. You will have to update `src/cli/ExampleProjectCommand.ts`. We might have to use alternatives to `rm` and `mv` commands also.

Reply:
Thanks @AdityaHegde , I did note that in the associated issue here. https://github.com/rilldata/rill-developer/issues/481#issuecomment-1179310653.  I am not sure I am the right person to finish the work, but I need to find the someone to help and I am not sure it will make it this sprint given our priorities.

Reply:
I ported `scripts/example-project.sh` into `src/cli/ExampleProjectCommand.ts`, so I'll mark this as resolved.